qa-portal-angular/projects/qa-cv/src/app/view-cv/view-cv.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="qa-app-title">
+<!-- <div class="qa-app-title">
```

Comment:
Cleanup these comments

Reply:
qa-portal-angular/projects/qa-cv/src/app/view-cv/view-cv.component.html
is deleted on this branch as it is not needed. 
I don't understand why this has appeared in the merge.